sdk/python/tests/integration/feature_repos/universal/data_sources/file.py

```diff
@@ -14,35 +14,46 @@
 
 
 class FileDataSourceCreator(DataSourceCreator):
-    f: Any
+    files: List[Any]
```

Comment:
Do we need to persist all of the files here? I don't understand how it's meant to be used. Aren't files created and a data source returned? Why do we need to keep it open? I can understand that the connection to the offline store or staging area should remain open, but not access to individual files.